app/controllers/results_controller.rb

```diff
@@ -13,8 +13,6 @@ def index
     begin
       @courses = @results_view.courses.all
       @number_of_courses_string = @results_view.number_of_courses_string
-
-      expires_in 10.minutes
```

Comment:
To keep the caching behaviour as simple as possible, remove the existing client-side caching we're doing here. This only works in a subset of browsers anyway (eg—Chrome ignores `max-age` a lot of the time).